nekoyume/Assets/_Scripts/State/LocalLayerModifier.cs

```diff
@@ -358,7 +336,7 @@ public static void AddReceivableQuest(Address avatarAddress, int id)
         public static void RemoveReceivableQuest(
             Address avatarAddress,
             int id,
-            bool resetState = true)
+            bool resetState = default)
```

Comment:
I know the bool default value is false, is this okay?

Reply:
Yes it is. I have fixed all other arguments.